src/components/Login/Login.js

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import GoogleLogin from 'react-google-login';
 import * as PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
 import { failLogin, successLogin } from '../../services/loginService';
 
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '100%',
+    height: '100%',
+    margin: 'auto 50%'
+  }
+}));
```

Comment:
Excellent! I have run this locally and it looks fine for the moment!

![Screenshot 2020-05-22 at 12 33 49](https://user-images.githubusercontent.com/27693622/82663934-91fb1e80-9c28-11ea-97e0-43462235ce87.png)